src/main/webapp/resources/js/pages/projects/share/ShareLayout.jsx

```diff
@@ -89,7 +97,14 @@ export function ShareLayout({ redirect }) {
       ) : (
         <Space direction="vertical" style={{ width: `100%` }}>
           <ShareProject currentProject={currentProject} />
-          <ShareSamples samples={samples} redirect={redirect} />
+          <Tabs>
+            <Tabs.TabPane tab={i18n("ShareLayout.samples")} key="samples">
+              <ShareSamples samples={samples} redirect={redirect} />
+            </Tabs.TabPane>
+            <Tabs.TabPane tab={i18n("ShareLayout.fields")} key="metadata">
```

Comment:
Just a thought. Should we maybe make the metadata fields portion of it into it's own step? It would make the user explicitly look at the fields and the target restrictions

Reply:
OUtdated, done